.github/workflows/ci.yml

```diff
@@ -48,21 +48,31 @@ jobs:
           show-versioninfo: true
       - uses: julia-actions/cache@v2
 
-      - name: Run tests
+      - name: Run tests without coverage
+        if: ${{ matrix.os != 'ubuntu-latest' }}
+        uses: julia-actions/julia-runtest@v1
+        with:
+          coverage: false
+
+      - name: Run tests with coverage
+        if: ${{ matrix.os == 'ubuntu-latest' }}
         uses: julia-actions/julia-runtest@v1
         with:
           coverage: true
 
       - name: Process Julia coverage data
+        if: ${{ matrix.os == 'ubuntu-latest' }}
         uses: julia-actions/julia-processcoverage@v1
 
       - name: Upload coverage data (Codecov)
+        if: ${{ matrix.os == 'ubuntu-latest' }}
         uses: codecov/codecov-action@v4
         with:
           files: lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
 
       - name: Upload coverage data (Coveralls)
```

Comment:
This will not be enough when you split the workload into multiple CI jobs like we do in Trixi.jl. Then, you would need to do something like we do over there. But this solution is fine for now

Reply:
Ok, I will try to keep this in mind.
Thanks for your attention here!